player/bundle/bundle.bzl

```diff
@@ -11,14 +11,14 @@ MODES = {
     DEV: DEVELOPMENT,
 }
 
-def webpack(name, dist, deps, mode_shorthand, visibility, bundle_name = None, **kwargs):
+def webpack(name, dist, deps, mode_shorthand, visibility, bundle_entry, bundle_name = None, **kwargs):
     output_name = bundle_name if bundle_name != None else name
     _webpack(
         name = "%s_bundle_%s" % (name, mode_shorthand),
         outs = ["dist/%s.%s.js" % (bundle_name, mode_shorthand)],
         args = [
             "--mode %s" % MODES[mode_shorthand],
-            "./$(RULEDIR)",
+            bundle_entry,
```

Comment:
Is this always necessary? Or can we default to something or maybe even only include this argument if defined?

Reply:
That's a good point, I guess there would be a use case for supplying the entry point from the webpack config. I'll make it an optional parameter.

Reply:
Should this entry be left up to the webpack config to decide?